projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,27 @@
 FROM python:3.10
+WORKDIR /app
 
+COPY . /app
+
+RUN pip install -r requirements.txt
+ENV "DATABASE_URL"="mongodb://mongo-example:27017"
+CMD ["python", "main.py"]
+
+EXPOSE 8000
+
+#Na ocenę od 3 do 4.0
+#docker build -t laby .
+#docker network create moze
+#docker run -d --network moze --name mongo_example -p 27017:27017 mongo:latest
+#docker run -d --network moze --name laby_sieci -p 8000:8000 a839a7d7d199
+#docker volume create laby_volume
+#docker volume create mongo_volume
+#Na ocenę 4.5 - dodane do wersji na 4.0 możliwość zawierania wolumenów
+#docker run -d --network moze -v laby_volume:/app --name laby_example -p 8000:8000 laby:latest
+#docker run -d --network moze -v mongo_volume:/app --name mongooo -p 27017:27017 mongo:latest
```

Comment:
W złe miejsce podłączyłeś volume. Miejsce gdzie trzymane są dane znajdziesz na stronie https://hub.docker.com/_/mongo w Rozdziale "Where to store data"

Reply:
O i do testów polecam trzymać się jednoznacznego nazewnictwa (chociażby żeby nie musieć zmieniać co chwila zmiennej `ENV` na przykład).